js/adapt-contrib-tutor.js

```diff
@@ -9,25 +9,28 @@ class Tutor extends Backbone.Controller {
   initialize() {
     this.listenTo(Adapt, {
       'componentView:postRender': this.onComponentViewPostRender,
-      'questionView:showFeedback': this.onQuestionViewShowFeedback
+      'questionView:showFeedback': this.onQuestionViewShowFeedback,
+      'buttonsView:postRender': this.onButtonsViewPostRender
```

Comment:
Previously `buttonsView:postRender` was only listened to when needed rather than listening to all events and returning early. Minor point, but isn't it more efficient to only use this as required instead?

Reply:
The cost is so negligible it not worth making the code confusing.